### PR TITLE
Fix Long2IP issue with PHP7.1

### PR DESCRIPTION
**Pull request by Tithugues**
Since PHP7.1, as long2ip expects an int, due to the strict_mode, PHP throws an error.

### CODE REVIEW

File: libraries/classes/Util.php [hunk 1]
```diff
@@ -4740,5 +4740,8 @@ public static function sortableTableHeader($title, $sort, $initialSortOrder = 'A
         $url = 'db_structure.php' . Url::getCommon($urlParams);
 
         return self::linkOrButton($url, $title . $orderImg, $orderLinkParams);
-    }
+    }public static function isInteger($input)
```
Comment (williamdes) [inline libraries/classes/Util.php]: Looks strange, must be a mistake

**Review by williamdes — APPROVED** [review]
LGTM, will merge and fix up the merge if you did not have tilt to do so
Thanks for your contribution @Tithugues!

File: libraries/classes/Util.php (hunk 1 above)
Comment (Tithugues) [reply libraries/classes/Util.php]: Sorry, I didn't check this before submitting.

File: libraries/classes/Plugins/Transformations/Output/Text_Plain_Binarytoip.php [hunk 2]
```diff
@@ -9,6 +9,7 @@
 namespace PhpMyAdmin\Plugins\Transformations\Output;
 
 use PhpMyAdmin\Plugins\TransformationsPlugin;
+use function hex2bin;
```
Comment (williamdes) [inline libraries/classes/Plugins/Transformations/Output/Text_Plain_Binarytoip.php]: NB: remove since it breaks php I no not remember the version

Comment (Tithugues) [reply libraries/classes/Plugins/Transformations/Output/Text_Plain_Binarytoip.php]: It should be fixed now!

**Comment by Tithugues** [comment]
> This commit is missing Signed-Off-By line to indicate that you agree with phpMyAdmin Developer's Certificate of Origin. Please check [contributing documentation](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) for more information.

:sob: I did the change only, so it didn't signed off the commit…

**Comment by williamdes** [comment]
:tada: You also fixed #14906 !

**Comment by williamdes** [comment]
Merged, thank you @Tithugues !

**Pull request merged**